### ISSUE TITLE
"Anonymous" students

### ISSUE THREAD

**Issue by magentanova** [jah2488/classroom#132]
In my student list, some of my students are identified by their actual names, most only by their email addresses. Others are identified by neither, and either I need to discern it from their github/medium link, or they're totally anonymous to me, like the student below:

![screen shot 2015-09-21 at 8 49 29 am](https://cloud.githubusercontent.com/assets/9850596/9993522/e34a5f0c-603d-11e5-9251-2314ce2c10bf.png)

Granted, this is probably a student who's no longer in the course, but here's a student who's clearly "valid": 

![screen shot 2015-09-21 at 8 53 28 am](https://cloud.githubusercontent.com/assets/9850596/9993604/416cefc8-603e-11e5-8b24-2c73600ea482.png)

I guess it's not mandatory that students fill in the name or email field when signing up? Maybe the page could at least show their username.

**Comment by kurtisnelson** [comment]
Correct, you can have a Student record without a corresponding User record.

**Comment by magentanova** [comment]
what i was thinking was that students shouldn't be allowed to create a profile without a name. i told mine to go back and add one if they hadn't, but this is still what my assignment view looks like: 

![screen shot 2015-10-14 at 9 00 41 am](https://cloud.githubusercontent.com/assets/9850596/10485976/793666de-7252-11e5-978d-c286def1862b.png)

with blank fields for three students.

**Comment by jah2488** [comment]
Thats pretty weird, it should fall back to their email if they haven't provided a name.

However, I agree, we should go ahead and make their name part of the sign up process.
